{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/constance_prometheus.py

```diff
@@ -0,0 +1,124 @@
+"""
+Export django-constance variables as Prometheus metrics.
+
+By default this module exports all constance variables which are of the ``int``,
+``float``, ``bool`` or ``decimal.Decimal`` types and not included in the
+``CONSTANCE_PROMETHEUS_BLACKLIST`` sequence.
+
+The variables will appear named ``constance_config_VARIABLE_KEY`` and represented
+as Gauge metrics.
+
+The ``CONSTANCE_PROMETHEUS_WHITELIST`` sequence may specify additional variables
+to be exported. Their values will be cast into ``str`` and appended to the metric
+name (e.g. ``VAR="foo bar"`` will become ``constance_config_VAR_foo_bar``) and
+get 1.0 value. Once changed, the old metric becomes 0.0 and new one will appear,
+allowing to present different values on a graph.
+
+"""
+import os
+import re
+
+from constance import config, settings as constance_settings
+from constance.signals import config_updated
+from decimal import Decimal
+from django.conf import settings
+from django.dispatch import receiver
+from prometheus_client import REGISTRY, CollectorRegistry, Gauge
+
+from . import metrics
+
+__all__ = ("Metric", "on_constance_config_updated", "export_config")
+
+
+def is_blacklisted(key):
+    return key in getattr(settings, "CONSTANCE_PROMETHEUS_BLACKLIST", [])
+
+
+def is_whitelisted(key):
+    return key in getattr(settings, "CONSTANCE_PROMETHEUS_WHITELIST", [])
+
+
+def gets_monitored(key, value):
+    return not is_blacklisted(key) and (
+        isinstance(value, (int, float, bool, Decimal)) or is_whitelisted(key)
+    )
+
+
+METRIC_NAME_FILTER = re.compile(r"\W", re.ASCII)
+
+
+class Metric:
+    prefix = "constance_config"
```

Comment:
```suggestion
    PREFIX = "constance_config"
```
as per [PEP8](https://peps.python.org/pep-0008/#constants)